src/modules/auth/webauthn/WebAuthnScreen.js

```diff
@@ -41,12 +40,26 @@ function WebAuthnCard(props) {
 	const [ isLoading, setIsLoading ] = useState(true);
 	const [ isSubmitting, setIsSubmitting ] = useState(false);
 	const [ authenticators, setAuthenticators ] = useState([]);
-	const [ editing, setEditing ] = useState(false);
+	const [ globalEditMode, setGlobalEditMode ] = useState(false);
+
+	const { handleSubmit, register, formState, setValue, resetField, getValues } = useForm();
+
+	const regName = register(
+		"name",
+		{
+			validate: {
+				emptyInput: value => (value && value.toString().length !== 0) || t("WebAuthnScreen|Key name cannot be empty!"),
+				startWithNumber: value => !(/^\d/).test(value) || t("WebAuthnScreen|Invalid format, resource cannot start with a number"),
```

Comment:
@Aringoaway If you do copypaste, dont make it obvious and check what have you copypasted. What `resource` are you checking here? Its a `authenticator name` if I am aware, innit? Also rewrite `cannot` to `can't`

Pls update it everywhere...